go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/aquasecurity/tracee
+
+go 1.15
+
+replace github.com/aquasecurity/tracee/libbpfgo => ./libbpfgo
+
+require (
+	github.com/aquasecurity/tracee/libbpfgo v0.0.0-00010101000000-000000000000
+	github.com/aquasecurity/tracee/tracee-rules v0.0.0-20210127115517-c43b1c3394ec // indirect
+	github.com/iovisor/gobpf v0.0.0-20200529092446-49b58e11a4b5
+	github.com/stretchr/testify v1.5.1
+	github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2
+	github.com/urfave/cli/v2 v2.3.0
+)
```

Comment:
I see top level tracee dir doesn't have a go.mod. Should we add go.mod and go.sum on the top level .gitignore file? https://github.com/aquasecurity/tracee/blob/main/.gitignore

Regardless, I'll remove this as it doesn't belong here.

Reply:
no.. root dir doesn't need a go.mod since it's not a module

Reply:
hmph - I thought my IDE had added it in but it doesn't seem to be the case. Oh well cruft anyways - I've removed it. https://github.com/aquasecurity/tracee/pull/477/commits/91e527ce78084fb47118c00061276e70d5814fa2

Reply:
also remove top level go.sum :)

Reply:
Are you talking about this? https://github.com/aquasecurity/tracee/pull/477/files#diff-bddf0b78668bac130d388e24bf8daf6fa0db604d2dc07a78aca6806985376f0c Or something else?

Reply:
yeah my bad